### PR TITLE
[Docs] adding alter column type - on disk changes

### PR DESCRIPTION
**Pull request by premkumr**
- Adding a feature highlight for GA

### CODE REVIEW

File: docs/content/preview/releases/release-notes/v2.20.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ docker pull yugabytedb/yugabyte:2.20.0.0-b76
 * Support for Transactional CDC.
 * Wait-on conflict concurrency control with pg_locks. This feature is especially important to users that have workloads with many multi-tablet statements, which may have multiple sessions contending for the same locks concurrently. Wait-on conflict concurrency control helps avoid a deadlock scenario as well as ensure fairness so a session is not starved by new sessions. [Early Access]
 * Catalog metadata caching to improve YSQL connection scalability and performance with faster warm up times. [Early access]
+* Expand `ALTER TABLE` capabilities by introducing support for `ALTER COLUMN TYPE` operations that involve on-disk changes. This allows a more extensive range of data type modifications, accommodating scenarios such as changing the type itself and not just increasing size. [GA]
```
Comment (aishwarya24) [inline docs/content/preview/releases/release-notes/v2.20.md:68]: Support for ALTER COLUMN TYPE  with on-disk changes. Expand ALTER TABLE capabilities with a broader range of data type modifications, including changes to the type itself, not just size increments. [GA]

File: docs/layouts/shortcodes/badge/ga.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<a class="yb-badge"
+  style="background: #3FC47E; color:#FFF; border: 0px solid #3FC47E;"
```
Comment (ddhodge) [inline docs/layouts/shortcodes/badge/ga.html]:
```suggestion
  style="background: #CDEFE1; color:#FFF; border: 0px solid #CDEFE1;"
```

File: docs/content/preview/releases/release-notes/v2.20.md [hunk 3]
```diff
@@ -63,8 +63,9 @@ docker pull yugabytedb/yugabyte:2.20.0.0-b76
...
-* Catalog metadata caching to improve YSQL connection scalability and performance with faster warm up times. [Early access]
+* Wait-on conflict concurrency control with pg_locks. This feature is especially important to users that have workloads with many multi-tablet statements, which may have multiple sessions contending for the same locks concurrently. Wait-on conflict concurrency control helps avoid a deadlock scenario as well as ensure fairness so a session is not starved by new sessions. {{<badge/ea>}}
+* Catalog metadata caching to improve YSQL connection scalability and performance with faster warm up times. {{<badge/ea>}}
+* Support for ALTER COLUMN TYPE with on-disk changes. Expand ALTER TABLE capabilities with a broader range of data type modifications, including changes to the type itself, not just size increments. {{<badge/ga>}}
```
Comment (ddhodge) [inline docs/content/preview/releases/release-notes/v2.20.md:68]:
```suggestion
* Support for ALTER COLUMN TYPE with on-disk changes. Expand ALTER TABLE capabilities with a broader range of data type modifications, including changes to the type itself, not just size increments.
```

File: docs/layouts/shortcodes/badge/ga.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<a class="yb-badge"
```
[the remark this answers is not in the document]
Comment (premkumr) [reply docs/layouts/shortcodes/badge/ga.html:1]: We might have to call GA out too. Let me confirm with release folks.

[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/layouts/shortcodes/badge/ga.html:1]: No harm in having a badge for unforseen circumstances; but in general, we can't put GA on every GA feature, that would be all of them. Highlighting everything highlights nothing.

[the remark this answers is not in the document]
Comment (premkumr) [reply docs/layouts/shortcodes/badge/ga.html:5]: But for a feature that was released in prior release as TP/EA, should we not call out when it goes GA ?

**Pull request merged**